src/idlenesswatcher.cpp

```diff
@@ -102,8 +104,19 @@ void IdlenessWatcher::setup()
     }
 }
 
-void IdlenessWatcher::timeoutReached(int identifier)
+void IdlenessWatcher::timeoutReached(int identifier,int /*msec*/)
 {
+    // check if disable Idleness when fullscreen is enabled
+    if (mPSettings.isDisableIdlenessWhenFullscreenEnabled()) {
+        WId w = KWindowSystem::activeWindow();
+        KWindowInfo info(w, NET::WMState);
+        if (info.hasState(NET::FullScreen)) {
+            qDebug() << "idleness timeout reached with fullscreen window. Postponing...";
```

Comment:
Please remove or comment out the qDebug() line above!

Reply:
Ok!